specs/altair/spec.yaml

```diff
@@ -0,0 +1,28 @@
+md_files:
+  - ../phase0/beacon-chain.md
+  - ../phase0/fork-choice.md
+  - ../phase0/validator.md
+  - ../phase0/weak-subjectivity.md
+  - beacon-chain.md
+  - bls.md
+  - fork.md
+  - validator.md
+  - p2p-interface.md
+  - sync-protocol.md
+
+py_prefix:
+  - ../phase0/prefix.py
+  - prefix.py
+
+py_suffix:
+  - ../phase0/suffix.py
+  - suffix.py
+
+overridden_functions:
+  - eth2_aggregate_pubkeys
+  - ceillog2
+  - floorlog2
+
+predefined_vars:
+  - FINALIZED_ROOT_INDEX
+  - NEXT_SYNC_COMMITTEE_INDEX
```

Comment:
`FINALIZED_ROOT_INDEX` and `NEXT_SYNC_COMMITTEE_INDEX` are firstly defined in prefix.yaml and then are listed here. IMHO it would be nice to only have to write them in one place.

How about setting the value here? For example
```
predefined_vars:
  - FINALIZED_ROOT_INDEX: GeneralizedIndex(105)
  - NEXT_SYNC_COMMITTEE_INDEX: GeneralizedIndex(55)
```

And then merge `predefined_vars` into the spec in `setup.py`?


---

Also, it would be nice to add some comments on why we have to predefine them. (due to dependency)